install/0000_50_machine-api-operator_03_machineset.crd.yaml

```diff
@@ -94,9 +94,25 @@ spec:
           - replicas
           type: object
   version: v1alpha1
+  additionalPrinterColumns:
+  - JSONPath: .spec.replicas
+    description: Desired Replicas
+    name: Desired
+    type: string
+  - JSONPath: .status.replicas
+    description: Current Replicas
+    name: Current
+    type: string
```

Comment:
Should these not be `type: integer`?

Reply:
Good point. The docs [1] says:
> If the value inside a CustomResource does not match the type specified for the column, the value is omitted. Use CustomResource validation to ensure that the value types are correct.

So wondering why is it printed even though string != int. Maybe a type cast that anything can be casted to string (e.g. `fmt.Sprintf("...%v", columnValue)`)? @frobware did you notice any difference?

[1] https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#type

Reply:
I didn't notice any difference. My choice was influenced by the replica set TableColumnDefinitions. But let me experiment with switching to integer. 

Currently, with `string`, I see: 

```console
$ kubectl get machinesets  --all-namespaces 
NAMESPACE   NAME                                             DESIRED   CURRENT   READY     AGE
test        amcdermo-aws-actuator-worker-machineset-69e4fb   3         3         3         13m
```

Reply:
I switched to integers and squashed commits in https://github.com/openshift/machine-api-operator/pull/148/commits/d49b90530c5acb6f4a6a81232b1f2052d0582523